.github/workflows/promptflow-sdk-pfs-e2e-test.yml

```diff
@@ -1,43 +1,31 @@
 name: promptflow-sdk-pfs-e2e-test
+
 on:
-  schedule:
-    - cron: "40 18 * * *" # Every day starting at 2:40 BJT
-  pull_request_target:
+  pull_request:
     paths:
       - src/promptflow/**
       - scripts/**
-      - '**promptflow-sdk-pfs-e2e-test.yml'
+      - .github/workflows/promptflow-sdk-pfs-e2e-test.yml
+
   workflow_dispatch:
+
 env:
   packageSetupType: promptflow_with_extra
   testWorkingDirectory: ${{ github.workspace }}/src/promptflow
   PYTHONPATH: ${{ github.workspace }}/src/promptflow
   IS_IN_CI_PIPELINE: "true"
+
 jobs:
-  authorize:
-    environment:
-      # forked prs from pull_request_target will be run in external environment, domain prs will be run in internal environment
-      ${{ github.event_name == 'pull_request_target' &&
-      github.event.pull_request.head.repo.full_name != github.repository &&
-      'external' || 'internal' }}
-    runs-on: ubuntu-latest
-    steps:
-      - run: true
   sdk_pfs_e2e_test:
-    needs: authorize
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
why macos

Reply:
offline synced, we expect pull_request to cover more